packages/modules/src/BatchedSessionRouterModule.ts

```diff
@@ -150,21 +154,21 @@ export class BatchedSessionRouterModule extends BaseValidationModule {
       const proof = this.sessionKeyManagerModule.merkleTree.getHexProof(
         ethers.utils.keccak256(leafDataHex) as unknown as Buffer
       )
-      proofs.push(proof)
+
+      sessionDataTuple.push(proof)
+      sessionDataTuple.push('0x')
```

Comment:
This extra data should also come from sessionSignerData

Reply:
you mean this '0x' ?
I will confirm with this @filmakarov if this is same as older additionalSig param used in session key manager module.

Reply:
Yes i think it's the additionalSig param, in BatchedSessionRouter this param is passed to SVM.

Reply:
ok changed it to this
sessionDataTuple.push(sessionParam.additionalSessionData ?? '0x')

it was already part of types